addons/website_slides/models/slide_channel.py

```diff
@@ -46,6 +47,38 @@ class ChannelUsersRelation(models.Model):
         )
     ]
 
+    def _compute_next_slide_id(self):
+        self.env['slide.channel.partner'].flush_model()
+        self.env['slide.slide'].flush_model()
+        self.env['slide.slide.partner'].flush_model()
+        query = """
+            SELECT DISTINCT ON (SCP.id)
```

Comment:
Note that this currently does not work:

![image](https://user-images.githubusercontent.com/41282536/224680570-d4b6b7c4-1b21-4e63-8ca9-ed31dcc82c83.png)

I think that's because it considers slides that are categories within the results.
(Would be nice to add a specific test case for that).